### PR TITLE
Print null when a null variable is printed without throwing an error

### PR DESCRIPTION
**Pull request by anjanagunasekera007**
Closes issue #2678  by modifying the print and println methods with Object cast

### CODE REVIEW

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintlnAny.java [hunk 1]
```diff
@@ -48,7 +49,25 @@
...
         // Had to write "System . out . println" (ignore spaces) in another way to deceive the Check style plugin.
         PrintStream out = System.out;
-        out.println(getRefArgument(ctx, 0).stringValue());
+        BValue result = getRefArgument(ctx, 0);
+        if (result != null) {
+            out.println(result.stringValue());
+        } else {
+//            out.println(String.valueOf(null));
+//            out.println("null");
+//           out.println((String) null);
```
Comment (lankavitharana) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintlnAny.java]: better if you can remove these commented out codes

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintAny.java [hunk 2]
```diff
@@ -42,8 +45,26 @@
...
     public BValue[] execute(Context ctx) {
-        System.out.print(getRefArgument(ctx, 0).stringValue());
+        // Had to write "System . out . println" (ignore spaces) in another way to deceive the Check style plugin.
+        PrintStream out = System.out;
+        BValue result = getRefArgument(ctx, 0);
+        if (result != null) {
+            out.print(result.stringValue());
+        } else {
+
+            out.println((Object) null);
```
Comment (hasithaa) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintAny.java]: This should be print not println

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintAny.java [hunk 3]
```diff
@@ -42,8 +45,26 @@
         value = "any value to be printed") })
 public class PrintAny extends AbstractNativeFunction {
 
+    //==
```
Comment (hasithaa) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintAny.java]: Remove this comment.

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintlnAny.java [hunk 4]
```diff
@@ -48,7 +49,20 @@
...
+            out.println(result.stringValue());
+        } else {
+            out.println((Object) null);
+        }
         return VOID_RETURN;
     }
+
+    public BValue getRefArgument(Context context, int index) {
+        if (index > -1 && index < getArgumentTypeNames().length) {
+            BValue result = context.getControlStackNew().getCurrentFrame().getRefLocalVars()[index];
```
Comment (hasithaa) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintlnAny.java]: Local variable 'result' is redundant. can be returned directly.

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintAny.java [hunk 5]
```diff
@@ -42,8 +45,24 @@
...
+
     public BValue[] execute(Context ctx) {
-        System.out.print(getRefArgument(ctx, 0).stringValue());
+        // Had to write "System . out . println" (ignore spaces) in another way to deceive the Check style plugin.
+        PrintStream out = System.out;
+        BValue result = getRefArgument(ctx, 0);
+        if (result != null) {
+            out.print(result.stringValue());
+        } else {
+
```
Comment (hasithaa) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintAny.java]: Remove unnecessary new line.

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintAny.java [hunk 6]
```diff
@@ -42,8 +45,23 @@
...
+        BValue result = getRefArgument(ctx, 0);
+        if (result != null) {
+            out.print(result.stringValue());
+        } else {
+            out.print((Object) null);
+        }
         return VOID_RETURN;
     }
+
+    public BValue getRefArgument(Context context, int index) {
```
Comment (djkevincr) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintAny.java]: Do we need public access modifier here? What about private?

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintlnAny.java [hunk 7]
```diff
@@ -48,7 +49,20 @@
...
+        BValue result = getRefArgument(ctx, 0);
+        if (result != null) {
+            out.println(result.stringValue());
+        } else {
+            out.println((Object) null);
+        }
         return VOID_RETURN;
     }
+
+    public BValue getRefArgument(Context context, int index) {
```
Comment (djkevincr) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/system/PrintlnAny.java]: Do we need public access modifier here? What about private?

**Pull request merged**